engines/e_capi_err.c

```diff
@@ -14,10 +14,10 @@
 #ifndef OPENSSL_NO_ERR
 
 static ERR_STRING_DATA CAPI_str_reasons[] = {
-    {ERR_PACK(0, 0, CAPI_R_CANT_CREATE_HASH_OBJECT), "cant create hash object"},
-    {ERR_PACK(0, 0, CAPI_R_CANT_FIND_CAPI_CONTEXT), "cant find capi context"},
-    {ERR_PACK(0, 0, CAPI_R_CANT_GET_KEY), "cant get key"},
-    {ERR_PACK(0, 0, CAPI_R_CANT_SET_HASH_VALUE), "cant set hash value"},
+    {ERR_PACK(0, 0, CAPI_R_CANT_CREATE_HASH_OBJECT), "can't create hash object"},
```

Comment:
This is a generated file..

Reply:
Yes, but it does need to be modified too. See https://github.com/openssl/openssl/pull/20910#discussion_r1189827045.